content/blog/how-to-get-started-with-net/index.md

```diff
@@ -0,0 +1,124 @@
+
+# Writing Code
+
+I know what you are probably thinking, why haven't we written any code yet? Well, the answer is that we haven't needed to. First we need to decide what
+we want out code to do. I will show you how to write a small program that will print a list of films. This will include the title, the year, and the director.
+
+We will create a new class called Film, and then we will create a new method called `PrintFilmInfo` which will print the title, year, and director of the film.
+We will also create a List of films, and then we will add a few films to the list.
+
+Lets get started with writing code:
+```cs
+using System;
+using System.Collections.Generic;
+
+namespace BoilerPlateConsoleApp
+{
+    class Film
+    {
+        public string Title { get; set; }
+        public int Year { get; set; }
+        public string Director { get; set; }
+
+        public void PrintFilmInfo()
+        {
+            Console.WriteLine($"Title: {Title}");
+            Console.WriteLine($"Year: {Year}");
+            Console.WriteLine($"Director: {Director}");
+        }
+    }
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            List<Film> films = new List<Film>();
+            films.Add(new Film() { Title = "The Shawshank Redemption", Year = 1994, Director = "Frank Darabont" });
+            films.Add(new Film() { Title = "The GodFather Part II", Year = 1974, Director = "Francis Ford Coppola" });
+            films.Add(new Film() { Title = "The Dark Night", Year = 2008, Director = "Christopher Nolan" });
+
+            foreach (var film in films)
+            {
+                film.PrintFilmInfo();
+            }
+        }
+    }
+}
+```
+
+This is the complete code for the film list. You can run the code to see the output. Might I also mention that I have used the `foreach` loop to print out
+the films. This is a very common loop in C#, and it is very easy to use. You can use it to loop through a list of items, and then print out the items.
+The `List<Film>` is a data structure that is used to store a list of items. The `Film` class is a class that we created to store information about a film.
+Inside of the `Film` class, we have three properties, `Title`, `Year`, and `Director`. Properties are like variables, and they are used to store information.
+
+I'm not going to go into nitty gritty details about what is going on here, as I expect you to be familiar with some of the concepts already. I really recommend
+reading up on the C# language, and reading the documentation, at https://docs.microsoft.com/dotnet/csharp to get a better understanding of what is going on here.
+
+I fully understand that dotnet can be a bit confusing at first, but I hope that this article will help you get started with .NET. I hope you enjoyed the article!
+
+# Contact
+
+If you have any questions, comments, or concerns, feel free to contact me at https://www.owenboreham.tech/contact or by email at contact@owenboreham.tech
```

Comment:
I would avoid having your email listed here. It's 100% guaranteed to be spammed with bots.

Even our Disqus comment system is spammed with a lot of bots lately 

```suggestion
If you have any questions, comments, or concerns, feel free to contact me at https://www.owenboreham.tech/contact or by email at contact@owenboreham.tech
```